ui/lib/apps/TopSQL/utils/telemetry.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 PingCAP, Inc. Licensed under Apache-2.0.
+import { mixpanel } from '@lib/utils/telemetry'
+import { TimeRange } from '@lib/components'
+import { TopsqlEditableConfig } from '@lib/client'
+
+export const telemetry = {
+  openSelectInstance() {
+    mixpanel.track('TopSQL: Open Select Instance')
+  },
+  finishSelectInstance(type: string) {
+    mixpanel.track('TopSQL: Finish Select Instance', { type })
+  },
```

Comment:
Will this leak user data?

Reply:
No, we only collect instance types instead of other instance information.